installer/bundle/bundle_skel_PPC-RHEL.sh

```diff
@@ -0,0 +1,584 @@
+        pkg_rm scx
+        pkg_rm omi
+    fi
+
+    if [ "$installMode" = "P" ]
+    then
+        echo "Purging all files in cross-platform agent ..."
+        rm -rf /etc/opt/microsoft/*-cimprov /etc/opt/microsoft/scx /opt/microsoft/*-cimprov /opt/microsoft/scx /var/opt/microsoft/*-cimprov /var/opt/microsoft/scx
+        rmdir /etc/opt/microsoft /opt/microsoft /var/opt/microsoft 1>/dev/null 2>/dev/null
+
+        # If OMI is not installed, purge its directories as well.
+        check_if_pkg_is_installed omi
+        if [ $? -ne 0 ]; then
+            rm -rf /etc/opt/omi /opt/omi /var/opt/omi
+        fi
+    fi
+fi
+
+if [ -n "${shouldexit}" ]
+then
+    # when extracting script/tarball don't also install
+    cleanup_and_exit 0
+fi
+
+#
+# Do stuff before extracting the binary here, for example test [ `id -u` -eq 0 ],
+# validate space, platform, uninstall a previous version, backup config data, etc...
+#
+
+#
+# Extract the binary here.
+#
+
+echo "Extracting..."
+tail -n +${SCRIPT_LEN_PLUS_ONE} "${SCRIPT}" | tar xzf -
+STATUS=$?
+if [ ${STATUS} -ne 0 ]
+then
+    echo "Failed: could not extract the install bundle."
+    cleanup_and_exit ${STATUS}
+fi
+
+#
+# Do stuff after extracting the binary here, such as actually installing the package.
+#
+
+EXIT_STATUS=0
+SCX_OMI_EXIT_STATUS=0
+BUNDLE_EXIT_STATUS=0
+
+case "$installMode" in
+    E)
+        # Files are extracted, so just exit
+        cleanup_and_exit 0 "SAVE"
+        ;;
+
+    I)
+        echo "Installing cross-platform agent ..."
+
+        if [ $PROVIDER_ONLY -eq 0 ]; then
+            check_if_pkg_is_installed omi
+            if [ $? -eq 0 ]; then
+                pkg_upd_list $OMI_PKG omi
+                pkg_upd ${UPD_PKG_QUEUE}
+            else
+                pkg_add_list $OMI_PKG omi
+            fi
+        fi
+
+        pkg_add_list $OM_PKG scx
+
+        pkg_add ${ADD_PKG_QUEUE}
+        SCX_OMI_EXIT_STATUS=$?
+
+        if [ $PROVIDER_ONLY -eq 0 ]; then
+            # Install bundled providers
+            [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+            for i in *-oss-test.sh; do
+                # If filespec didn't expand, break out of loop
+                [ ! -f $i ] && break
+                ./$i
+                if [ $? -eq 0 ]; then
+                    OSS_BUNDLE=`basename $i -oss-test.sh`
+                    ./${OSS_BUNDLE}-cimprov-*.sh --install $FORCE $restartDependencies
+                    TEMP_STATUS=$?
+                    [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
+                fi
+            done
+        fi
+        ;;
+
+    U)
+        echo "Updating cross-platform agent ..."
+        if [ $PROVIDER_ONLY -eq 0 ]; then
+            shouldInstall_omi
+            pkg_upd_list $OMI_PKG omi $?
+        fi
+
+        shouldInstall_scx
+        pkg_upd_list $OM_PKG scx $?
+
+        pkg_upd ${UPD_PKG_QUEUE}
+        SCX_OMI_EXIT_STATUS=$?
+
+        if [ $PROVIDER_ONLY -eq 0 ]; then
```

Comment:
As per comments above, delete 553-568. In fact, I think the entire notion of "PROVIDER_ONLY" falls out on this platform, just get rid of all notions of that. Thanks.